.github/workflows/ci.yml

```diff
@@ -340,7 +340,7 @@ jobs:
           make install > /dev/null
 
       - name: 22.2 Unit Testing
-        run: make unittest
+        run: make unittest-dev-222
```

Comment:
Is there a reason to have a specific versioned name in this case? It would be nice if the generic non-versioned name was still there and executes the latest supported release.

Reply:
`make unittest` will run the default set for users which is dev unittests with 22.2 image (we will change that to 23.1 image after 23.1 Fluent release). As we are running both in CI, explicitly added the version for clarity.